guides/doc-Planning_for_Project/topics/Glossary.adoc

```diff
@@ -244,6 +244,8 @@ You can promote a content view through a typical promotion path; for example, fr
 *Manifest (Red{nbsp}Hat Subscription Manifest)*:: A mechanism for transferring subscriptions from the Red{nbsp}Hat Customer Portal to {ProjectName}.
 Do not confuse with xref:varl-Glossary_of_Terms-Puppet_Manifest[Puppet Manifest].
 
+[[varl-Glossary_of_Terms-Migration]]
+*Migration*:: The process of moving an existing {Project} installation to a new instance.
```

Comment:
```suggestion
[[varl-Glossary_of_Terms-Migrating]]
*Migrating {Project}*:: The process of moving an existing {Project} installation to a new instance.
```
Because other kinds of migrations exist, we want to specify that we mean the whole project.

Reply:
Agree :+1: